### PR TITLE
GitHub repo

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -9,13 +9,28 @@
...
-ynh_script_progression --message="Setting up source files..." --weight=4
+ynh_script_progression --message="Setting up source files..."
+
 
-# Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir="$install_dir"
+tmpdir=$(mktemp -d)
+ynh_setup_source --dest_dir="$tmpdir"
+pushd $tmpdir
+    npm ci
```
Comment (alexAubin) [inline scripts/install:29]:
```suggestion
pushd $tmpdir
    ls -l
    npm ci
```

File: manifest.toml [hunk 2]
```diff
@@ -43,8 +43,8 @@ ram.runtime = "50M"
     [resources.sources]
 
         [resources.sources.main]
-        url = "https://release.larsjung.de/h5ai/h5ai-0.30.0.zip"
-        sha256 = "fe138305a61d72094beb8c507dab728020158e4b00dae695a60b9803babb3e30"
+        url = "https://github.com/lrsjng/h5ai/archive/refs/tags/v0.30.0.tar.gz"
+        sha256 = "f1638557ec79e6fe6bd9303b1e9d8d0180a620dbf4811b6c194befa24769800c"
         in_subdir = false
```
Comment (alexAubin) [inline manifest.toml]:
```suggestion
```

File: scripts/install [hunk 3]
```diff
@@ -9,13 +9,29 @@
...
-ynh_script_progression --message="Setting up source files..." --weight=4
+ynh_script_progression --message="Setting up source files..."
+
 
-# Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir="$install_dir"
+tmpdir=$(mktemp -d)
+ynh_setup_source --dest_dir="$tmpdir"
+pushd $tmpdir
+    ls -l
```
Comment (alexAubin) [inline scripts/install]:
```suggestion
```

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**